src/os/i18n.zig

```diff
@@ -26,6 +26,7 @@ const log = std.log.scoped(.i18n);
 pub const locales = [_][:0]const u8{
     "de_DE.UTF-8",
     "zh_CN.UTF-8",
+    "uk_UA.UTF-8",
```

Comment:
I'd recommend keeping these sorted according to their ASCII ordering.

Reply:
Actually the order does matter. See the doc comment above